pages/[slug]/login.tsx

```diff
@@ -0,0 +1,117 @@
+import { NextPage } from 'next'
+import Image from 'next/image'
+import Link from 'next/link'
+import logo from '../../public/devpleno.svg'
+import * as Yup from 'yup'
+import { useFormik } from 'formik'
+
+const LoginSchema = Yup.object().shape({
+  email: Yup.string().email('Email inválido').required('Campo obrigatório'),
+  password: Yup.string().required('Campo obrigatório'),
+})
+const Login: NextPage = () => {
+  const form = useFormik({
+    validateOnChange: false,
+    validateOnMount: false,
+    validateOnBlur: true,
+    initialValues: {
+      email: '',
+      password: '',
+    },
+    validationSchema: LoginSchema,
+    onSubmit: async values => {
+      console.log(values)
+    },
+  })
+  return (
+    <div className={`min-h-screen bg-gray-50 dark:bg-gray-900`}>
+      <div className='flex flex-col md:flex-row h-screen overflow-y-auto'>
+        <div className=' flex h-64 md:h-auto w-full md:w-1/2 bg-initial bg-cover bg-center'></div>
+        <main className='flex justify-center items-center md:w-1/2 p-6 sm:px-8 '>
+          <div className='w-full max-w-lg'>
+            <h1 className='mb-4 text-xl font-semibold text-gray-600 dark:text-gray-200 capitalize'>
+              friendlyName

```

Comment:
Replace this "friendlyName" to dynamic data (this value has appeared on top of page)